docs/book/configuration.md

```diff
@@ -81,6 +82,7 @@ multiple services.
 | Field | Type | Required | Description |
 | --- | --- | --- | --- |
 | `bundle.name` | `string` | Yes | Name of the bundle to download. |
+| `bundle.prefix` | `string` | No | Prefix to add to the name of the bundle |
```

Comment:
`No (default: bundles)` . Also the [Bundle docs](https://www.openpolicyagent.org/docs/bundles.html) need to be updated with the `prefix` explanation similar to the [Discovery docs](https://www.openpolicyagent.org/docs/discovery.html).